### PR TITLE
fix(prometheus): Disable authorization for metrics scraping endpoint

### PR DESCRIPTION
**Pull request by ieQu1**

### CODE REVIEW

File: apps/emqx_authz/src/emqx_authz.appup.src [hunk 1]
```diff
@@ -1,8 +1,13 @@
+%% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  [{"0.1.1",
-    [{load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
+  [{"0.1.0",[{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]}]},
+   {"0.1.1",
+    [{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]},
+     {load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
```
Comment (zmstone) [inline apps/emqx_authz/src/emqx_authz.appup.src:7]: 0.1.1 is not released yet.

Comment (ieQu1) [reply apps/emqx_authz/src/emqx_authz.appup.src]: But it was wrongly bumped in the app src file, so the script filled in all the gaps, assuming the internal applications follow semver. It's too time consuming to fight this.

File: apps/emqx_authz/src/emqx_authz.appup.src [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  [{"0.1.1",
-    [{load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
-   {<<".*">>,[]}],
-  [{"0.1.1",
-    [{load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
-   {<<".*">>,[]}]}.
+  [{"0.1.0",[{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]}]},
+   {"0.1.1",[{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]}]}],
```
Comment (zmstone) [inline apps/emqx_authz/src/emqx_authz.appup.src:5]: 0.1.1 is not released yet.

**Review by zmstone — CHANGES_REQUESTED** [review]

File: apps/emqx_authz/src/emqx_authz.appup.src (hunk 2 above)
Comment (zmstone) [reply apps/emqx_authz/src/emqx_authz.appup.src:5]: Please help to revert the change

Comment (zmstone) [reply apps/emqx_authz/src/emqx_authz.appup.src:5]: Wonder why the version jump was not caught by:
https://github.com/emqx/emqx/blob/466fd52168f78c967150f9e4d004995daf63c978/scripts/apps-version-check.sh#L44

Comment (ieQu1) [reply apps/emqx_authz/src/emqx_authz.appup.src:5]: The problem is, it has been bumped before 5.0.0 release. This is the version from 5.0.0:

```
{application, emqx_authz, [
    {description, "An OTP application"},
    {vsn, "0.1.1"},
    {registered, []},
    {mod, {emqx_authz_app, []}},
    {applications, [
        kernel,
        stdlib,
        crypto,
        emqx_connector
    ]},
    {env, []},
    {modules, []},

    {licenses, ["Apache 2.0"]},
    {links, []}
]}.
```

So in 5.0.1 the version should be bumped, and the appup should take care of 0.1.1

**Review by zmstone — APPROVED** [review]

**Pull request merged**